build/data/schema.json

```diff
@@ -59,6 +59,9 @@
       "additionalProperties": {
         "type": "object",
         "properties": {
+          "capital": {
+            "type": "boolean"
```

Comment:
From my point of view this property should be required, otherwise it'll be error prone.

Reply:
I made it optional because it's pretty easy to detect the case of the character. If we made it required, we'd have to update all the `src` files which seems unnecessary to me.